packages/mapviewer/src/modules/menu/components/advancedTools/MenuAdvancedToolsList.vue

```diff
@@ -23,6 +25,12 @@ const isCompareSliderActive = computed(() => store.state.ui.isCompareSliderActiv
 const isPhoneMode = computed(() => store.getters.isPhoneMode)
 const is3dActive = computed(() => store.state.cesium.active)
 
+const { t } = useI18n()
+
+const hasNoActiveLayersWarning = computed(() => {
+    return !isCompareSliderActive.value && store.getters.visibleLayers.length === 0
```

Comment:
you can simply use `const hasNoVisibleLayer = computed(() => !store.getters.visibleLayerOnTop)`

also, be careful with the naming convention : we are not checking the active layers, only the visible one